lib/msal-browser/src/cache/BrowserStorage.ts

```diff
@@ -420,4 +420,11 @@ export class BrowserStorage extends CacheManager {
             throw BrowserAuthError.createTokenRequestCacheError(err);
         }
     }
+
+    /**
+     * Gets the clientId
+     */
+    getClientId(): string {
```

Comment:
I still don't love this being part of the Storage class since it doesnt really have anything to do with storage. Would rather the clientId get passed into the RedirectHandler if thats the only place this is used

Reply:
Done